sdk/src/ordinal-api/index.ts

```diff
@@ -432,8 +432,8 @@ export class OrdinalsClient {
      * console.log("Output:", output);
      * ```
      */
-    async getInscriptionsFromOutPoint(outPoint: OutPoint): Promise<OutputJson> {
-        return await this.getJson<OutputJson>(`${this.basePath}/output/${OutPoint.toString(outPoint)}`);
+    getInscriptionsFromOutPoint(outPoint: OutPoint): Promise<OutputJson> {
```

Comment:
no need to have async/await here. not sure if runtime mitigates that nowadays but adding more generator wrappers (async/await) puts argument callback back to the end of the microtask queue and delays execution